Assignments/SCF_Session-2/SCF_Assignment-1/src/HexCalculator.java

```diff
@@ -0,0 +1,122 @@
+
+public class HexCalculator implements HexCalcMethods{
+
+	@Override
+	public String add(String firstNumber, String secondNumber) // adds two hex numbers and returns hexnumber 
+	{
+		int sum=0;
+		int number1=Integer.parseInt(hexToDecimal(firstNumber));
+		int number2=Integer.parseInt(hexToDecimal(secondNumber));
+		sum=number1+number2;
+		
+		return (decimalToHex(String.valueOf(sum)));
+	}
+
+	@Override
+	public String subtract(String firstNumber, String secondNumber) // subtracts two hex numbers and returns hexnumber 
+	{
+		int difference=0;
+		int number1=Integer.parseInt(hexToDecimal(firstNumber));
+		int number2=Integer.parseInt(hexToDecimal(secondNumber));
+		difference=number1+number2;
+		
+		return (decimalToHex(String.valueOf(difference)));
+	}
+
+	@Override
+	public String multiply(String firstNumber, String secondNumber) // multiply two hex numbers and returns hexnumber 
+	{
+		int product=0;
+		int number1=Integer.parseInt(hexToDecimal(firstNumber));
+		int number2=Integer.parseInt(hexToDecimal(secondNumber));
+		product=number1+number2;
+		
+		return (decimalToHex(String.valueOf(product)));
+	}
+
+	@Override
+	public String divide(String firstNumber, String secondNumber) // divide two hex numbers and returns hexnumber 
+	{
+		int divideResult=0;
+		int number1=Integer.parseInt(hexToDecimal(firstNumber));
+		int number2=Integer.parseInt(hexToDecimal(secondNumber));
+		divideResult=number1+number2;
+		
+		return (decimalToHex(String.valueOf(divideResult)));
+	}
+
+	@Override
+	public boolean equal(String firstNumber, String secondNumber) //compare two hex numbers
+	{
+		boolean flag=false; 
+		if(firstNumber.compareTo(secondNumber)==0){
+			flag=true;
+		}
+		else{
+			flag=false;
+		}
+		return flag;
+	}
+
+	@Override
+	public boolean greaterThan(String firstNumber, String secondNumber) //compare two hex numbers
+	{
+		boolean flag=false;
+		
+		if(firstNumber.compareTo(secondNumber)>0){
+			flag=true;
+		}
+		else{
+			flag=false;
+		}
+		return flag;
+	}
+
+	@Override
+	public boolean lessThan(String firstNumber, String secondNumber) //compare two hex numbers
```

Comment:
Method naming convention in a such way that the method could represent itselft a boolean type.